helpers/pointer.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"time"
+)
+
+func BoolP(b bool) *bool {
```

Comment:
I think the AWS package provides these, which we have used elsewhere, we should pick one or the other not both.